src/contributors/alsd4git.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "Alessandro Digilio",
+  "desc": "full stack web and android dev",
+  "github": "https://github.com/alsd4git",
+  "twitter": "https://twitter.com/alsd4tweet",
```

```suggestion
  "twitter": "https://twitter.com/alsd4tweet"
```
Remove this trailing-comma. This is what made some checks fail.